nf_core/__main__.py

```diff
@@ -398,16 +398,18 @@ def list(ctx):
 @click.pass_context
 @click.argument("keywords", required=False, nargs=-1, metavar="<filter keywords>")
 @click.option("-j", "--json", is_flag=True, help="Print as JSON to stdout")
-def remote(ctx, keywords, json):
+@click.option("-u", "--url", help="Remote repository url to list from")
+@click.option("-b", "--branch", help="Remote repository branch to list")
+def remote(ctx, keywords, json, url, branch):
     """
     List modules in a remote GitHub repo [dim i](e.g [link=https://github.com/nf-core/modules]nf-core/modules[/])[/].
     """
     try:
         module_list = nf_core.modules.ModuleList(
             None,
             True,
-            ctx.obj["modules_repo_url"],
-            ctx.obj["modules_repo_branch"],
+            ctx.obj["modules_repo_branch"] if url is None else url,
```

Comment:
Just checking that this isn't a copy/paste typo - could it be that it's meant to be this? 👇🏻 
```suggestion
            ctx.obj["modules_repo_url"] if url is None else url,
```